policybot/handlers/githubwebhook/filters/resulttester/resultTester.go

```diff
@@ -30,10 +30,10 @@ import (
 
 // Updates the DB based on incoming GitHub webhook events.
 type ResultTester struct {
```

Comment:
Sorry I'm so picky about names, but symbol names must convey  what they're about, otherwise folks trying to read and understand the  code get confused.

So  this handler is not a ResultTester, right? It's a handler that triggers the TestResultGatherer logic,  correct? So maybe just go  with handler/resultgatherer, as a partner to pkg/resultgatherer